### PR TITLE
#11: Improve Contributing file structure

### PR DESCRIPTION
**Pull request by mjecov**

### CODE REVIEW

**Review by katridi — DISMISSED** [review]
Seems good, later we will update js, css and html guides

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,3 +38,21 @@
...
 
 9. Any proposals can and should be voiced, considered and debated. When the decision is taken,  we are moving accordingly without deviation, except something previously unknown affects on the context in which the decision was taken. In such cases, we reconsider the decision. It happens and that is totally fine!
+
+## What should I know before I get started?
+
+## How Can I Contribute
+
+## Styleguides
+### Python Styleguide
+* We use [PEP-8](https://www.python.org/dev/peps/pep-0008/) in out project.
```
Comment (katridi) [inline CONTRIBUTING.md]: "in our project", right?

**Review by katridi — DISMISSED** [review]
Seems fine!

**Review by katridi — APPROVED** [review]

**Review by AndreiDrang — APPROVED** [review]
All is ok, gj

**Pull request merged**